Segment-Nielsen-DTVR/Classes/SEGNielsenDTVRIntegration.m

```diff
@@ -6,7 +6,7 @@
 //
 
 #import "SEGNielsenDTVRIntegration.h"
-#import "SEGTrackPayload.h"
+#import <Segment/SEGTrackPayload.h>
```

Comment:
To support everything, this should actually be:

```
#if __has_include(<Segment/SEGTrackPayload.h>)
#import <Segment/SEGTrackPayload.h>
#else 
#import "SEGTrackPayload.h"
#endif
```

Reply:
@bsneed I can make this change :+1: I'm curious why do you need to add this __has_include check? Also I noticed in other locations we have an extra check `#if defined(__has_include) && __has_include(*)`. Do you want that here too?

Reply:
The __has_include check lets us operate in different environments.  Things like cocoa pods, Carthage, react-native, spm all have their oddities about how they reference other modules and whatnot.  The defined(__has_include), it's your call on whether to add that or not, but it's been in ObjC for a really really long time now, so it's extremely unlikely that it would ever be false.

Reply:
Updated! Thanks for all the info

Reply:
No problem at all, anytime!  Thanks for your contributions!